staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -110,6 +138,12 @@ type byPriority struct {
 func (a byPriority) Len() int      { return len(a.specs) }
 func (a byPriority) Swap(i, j int) { a.specs[i], a.specs[j] = a.specs[j], a.specs[i] }
 func (a byPriority) Less(i, j int) bool {
+	// All local specs will come first
+	// WARNING: This will result in not following priorities for local APIServices.
```

Comment:
If we want this, this will be part of a follow-up. For now this looks good enough.